packages/table/components/Table.tsx

```diff
@@ -308,15 +319,16 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
     const updateHoveredRowIndex = () => {
       this.setState({ hoveredRowIndex: rowIndex });
     };
-    const selectedRows = this.props.selectedRows || [];
+    const selectedRows = this.props.selectedRows || {};
```

Comment:
Maybe we should add default props for `selectedRows` and `uniqueKey`?

https://github.com/sw-yx/react-typescript-cheatsheet#typing-defaultprops

Reply:
I get a Typescript error if I change these to default props